src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -287,11 +287,26 @@ export abstract class BaseWindow extends Disposable implements IBaseWindow {
 		return !!this.documentEdited;
 	}
 
-	focus(options?: { force: boolean }): void {
-		if (isMacintosh && options?.force) {
+	focus(options?: { force: boolean; notify: boolean }): void {
+		const hasAnyFocusedWindow = electron.BrowserWindow.getAllWindows().some(w => w.isFocused());
+
+		if (hasAnyFocusedWindow) {
```

Comment:
This new check changes the semantic of how `force: true` was handled and is imho risky to do because we do not know if `hasAnyFocusedWindow` is doing what we want.

Reply:
This was intentional. I observed that on macOS, simply calling `win.focus()` stole focus with other apps (it was basically always `force: true`). This change was made to avoid that behavior.